src-plugins/manualRegistration/manualRegistration.h

```diff
@@ -92,6 +91,10 @@ class MANUALREGISTRATIONPLUGIN_EXPORT manualRegistration : public itkProcessRegi
     */
     virtual bool writeTransform(const QString& file);
 
+    void setParameter(int data, int channel);
```

Comment:
Since there is only one channel here, I think you can set channel to 0 by default.